src/client/main.tsx

```diff
@@ -3,6 +3,31 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import { installDash } from './components/dash/install';
 
+import sio from 'socket.io-client';
+
+const uri = `${document.location!.origin}`;
+const sioNetwork = sio(uri, {
+  upgrade: false,
+  transports: ['websocket'],
+});
+sioNetwork.on('connect', () => {
+  console.log('Client connected to server');
+});
+sioNetwork.on('reconnect_attempt', () => {
+  console.log('Client reconnecting to server');
+  sioNetwork.io.opts.transports = ['polling', 'websocket'];
```

Comment:
Didn't we set up NUsight to be WebSocket only? We're unlikely to encounter a browser that doesn't support it these days.